app/helpers/validation_helper.rb

```diff
@@ -92,10 +92,26 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     monitored_address_state: VALID_STATES
   }.freeze
 
+  VALID_EXPOSURE_ENUMS = {
+    case_status: ['Suspect', 'Unknown', 'Not a Case']
+  }.freeze
+
+  VALID_ISOLATION_ENUMS = {
+    case_status: %w[Confirmed Probable]
+  }.freeze
+
   NORMALIZED_ENUMS = VALID_ENUMS.transform_values do |values|
     Hash[values.collect { |value| [value.to_s.downcase.gsub(/[ -.]/, ''), value] }]
   end
 
+  NORMALIZED_EXPOSURE_ENUMS = VALID_EXPOSURE_ENUMS.transform_values do |values|
+    Hash[values.collect { |value| [value.to_s.downcase.gsub(/[ -.]/, ''), value] }]
```

Comment:
Given we have this same line in three places now, it probably makes more sense to just break it out into a helper function.

Reply:
Just pushed an update addressing your suggestion.